### PR TITLE
Typographical and Grammatical Changes on intro page

### LINKED ISSUE #357: Bug: Grammatical Error and Possible Typo in Get-Started section
URL : http://hood.ie/intro/#get-started

Screenshot 2019-03-24 at 1 08 26 PM

the line `you who build something...` should be read as `you who builds something on it, don't have to worry about backends`

so basically change `build` -> `builds` and add a comma before `don't` to separate the conjuncture in the two sentences there 

Seems like an easy fix and I would like to implement this as my first issue here. 🥇

### PR DESCRIPTION
**Pull request by theNaut1lus**
Made some typographical and grammatical changes to the intro page
url: http://hood.ie/intro/#get-started
I have added my info in community page as well.
Fixes #357

### CODE REVIEW

File: intro/index.html [hunk 1]
```diff
@@ -136,7 +136,7 @@ <h1 class="h2">Hoodie runs locally and can be hosted</h1>
     <article class="grid-9">
         <h1 class="h2">Hoodie is Dreamcode based on an Offline First and noBackend architecture</h1>
         <p>
-            Hoodie is <a href="/initiatives#offline-first">Offline First</a>. Thus, your application users’ data is stored locally and accessible regardless of internet connection. It’s a <a href="/initiatives#nobackend">noBackend</a> architecture which means that you who build something on it don’t have to worry about backends, and its API is written in <a href="/initiatives#dreamcode">Dreamcode</a> so that frontend developers can easily understand it.
+            Hoodie is <a href="/initiatives#offline-first">Offline First</a>. Thus, your application users’ data is stored locally and accessible regardless of internet connection. It’s a <a href="/initiatives#nobackend">noBackend</a> architecture which means that you who builds something on it, don't have to worry about backends, and its API is written in <a href="/initiatives#dreamcode">Dreamcode</a> so that frontend developers can easily understand it.
```
Comment (gr2m) [inline intro/index.html:139]: 👍

File: community/index.html [hunk 2]
```diff
@@ -421,6 +421,11 @@ <h1 class="h2">
                     <img class="contribute-topic-image" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/jens-nikolaus.jpg" width="170" height="170" alt="Portrait photo(black and white) of Jens Nikolaus.">
                 </a>
             </div>
+            <div class="grid-3 l">
+                <a href="https://github.com/co16353sidak" target="_blank">
+                    <img class="contribute-topic-image" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/sidak-singh.jpg" width="170" height="170" alt="Portrait photo(black and white) of Sidak Singh Aulakh.">
+                </a>
+            </div>
```
Comment (gr2m) [inline community/index.html]: I’m sorry Sidak, but we only added the core team to this list. Hoodie had far more than 100 contributors, the people listed here are only a subset, with a lot of contributions :) So please remove that change

**Review by gr2m — CHANGES_REQUESTED** [review]

File: community/index.html (hunk 2 above)
Comment (theNaut1lus) [reply community/index.html]: Let me revert this right away

**Comment by theNaut1lus** [comment]
Hopefully now everything will be in order @gr2m

**Review by gr2m — APPROVED** [review]
Looks great, thank you Sidak!

**Pull request merged**